src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -231,6 +231,7 @@ export const SelectRegistrationTypeField = () => {
         </Box>
 
         <CategorySelector
+          enableNviHighlightning={true}
```

Comment:
Når man velger en kategori for et resultat kan man bare oppgi en verdi, så å velge alle der bør føre til en feil.

```suggestion
```

https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/c102af80-31f5-4432-b7d1-89962773c121